docs/contributing/translations.md

```diff
@@ -0,0 +1,36 @@
+# Translations
+Wagtail uses Transifex to translate the content for the admin interface. Our goal is to ensure that Wagtail can be used by those who speak many different languages. Translation of admin content is a great way to contribute without needing to know how to write code.
+
+## Translation workflow 
+Translations from Transifex are only integrated into the Wagtail repository at the time of a new release. When a release is close to being ready there will be a RC (Release Candidate) for the upcoming version and the translations will be exported to Transifex.
+
+During this RC period, usually around two weeks, there will be a chance for all the translators to update and add new translations. We will also notify the `#translators` channel in the Watail slack group at this time.
+
+These new translations are imported into Wagtail for any subsequent RC and the final release.
+
+If translations reach a threshold of about 80% we communicate that this is an officially supported translation in the docs.
```

Comment:
I don't think there's anything in the docs that lists the 'official' translations. Languages over 90% completion are added to the default list of languages that users can choose from, though - see [WAGTAIL_PERMITTED_LANGUAGES](https://docs.wagtail.org/en/stable/reference/settings.html#wagtailadmin-permitted-languages)

Reply:
Hi @gasman, I think we are both trying to drive the same message across that is if the translation  of a particular language is about 80% the language will be added .

Reply:
Yes, but we add it to the list inside Wagtail itself ([here, to be specific](https://github.com/wagtail/wagtail/blob/df0b36fdb7871b9e32cebdbd8be3f82022139c7c/wagtail/admin/localization.py#L8)...) - we don't communicate the list of supported languages anywhere in the docs.

Reply:
Oh okay.
So it should be "If translations reach a threshold of about 80% , languages are added to the default list of languages users can choose from." ?